.github/workflows/test.yaml

```diff
@@ -139,11 +140,32 @@ jobs:
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
       - name: Save hypothesis directory
+        uses: actions/cache/save@v4
+        if: always()
+        with:
+          path: .hypothesis/
+          key: cache-hypothesis-${{ github.run_id }}-${{ github.run_attempt }}-${{ matrix.environment }}-${{ matrix.os }}
```

Comment:
sorry i forgot why this was needed and deleted it when porting to datashader.

This is what we do in xarray
https://github.com/pydata/xarray/blob/ccb34374564de32c8df305285a7233dd892cdce5/.github/workflows/hypothesis.yaml#L98-L105